homeassistant/components/glances/sensor.py

```diff
@@ -237,6 +251,13 @@ class GlancesSensorEntityDescription(
         icon="mdi:harddisk",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ("raid", "config"): GlancesSensorEntityDescription(
+        key="config",
+        type="raid",
+        name_suffix="Raid configuration",
+        icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This seems incorrect, the example/test playload seems to be: `UU`, however, this is marked as a measurement?